consul/resource_service_defaults_config_entry_ce_test.go

```diff
@@ -0,0 +1,310 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
What is the difference between `ce_test` and `ee_test`?

Reply:
namespace partition is added.

Reply:
also it runs on enterprise binary